### PR TITLE
adding implementation of https://arxiv.org/abs/1805.06066

### PR DESCRIPTION
**Pull request by arsalan-mousavian**

### CODE REVIEW

**Comment by robieta** [comment]
Thanks @arsalan-mousavian. Typically, models in the Research repository require a collaborator at Google Research to take ownership. Are you working with someone there, or do you have somebody who is willing to act as reviewer and owner of the code?

**Comment by arsalan-mousavian** [comment]
This is the work I did last year in Brain Robotics. I think @ayzaan @mfiser or @toshev can review this PR. I am not sure if they are on the brain-github team as well.

**Comment by robieta** [comment]
Lovely. Thanks for letting me know.

File: research/cognitive_planning/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+## 2. Datasets
+
+### Download ActiveVision Dataset 
+We used Active Vision Dataset (AVD) which can be downloaded from [here](http://cs.unc.edu/~ammirato/active_vision_dataset_website/). To make our code faster and reduce memory footprint, we created the AVD Minimal dataset. AVD Minimal consists of low resolution images from the original AVD dataset. In addition, we added annotations for target views, predicted object detections from pre-trained object detector on MS-COCO dataset, and predicted semantic segmentation from pre-trained model on NYU-v2 dataset. AVD minimal can be downloaded from [here](https://storage.googleapis.com/active-vision-dataset/AVD_Minimal.zip). Set `$AVD_DIR` as the path to the downloaded AVD Minimal.
+
+### TODO: SUNCG Dataset
+Current version of the code does not support SUNCG dataset. It can be added by
+implementing necessary functions of `envs/task_env.py` using the public
+releaseed code of SUNCG environment such as
```
Comment (ayzaan) [inline research/cognitive_planning/README.md]: released

File: research/cognitive_planning/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+python viz_active_vision_dataset_main -- \
+  --mode=human \
+  --gin_config=envs/configs/active_vision_config.gin \
+  --gin_params='ActiveVisionDatasetEnv.dataset_root=$AVD_DIR'
+```
+
+## 3. Training
+Right now, the released version only supports training and inference using the real data from Active Vision Dataset.
+
+When RGB image modality is used Resnet the embedding is initialized  is used To start the training download pre-trained Resnet50 check point in the working directory ./resnet_v2_50_checkpoint/resnet_v2_50.ckpt
```
Comment (ayzaan) [inline research/cognitive_planning/README.md]: seems like a typo here.
Maybe something like:
When the RGB image modality is used, the Resnet embeddings are initialized.

**Comment by arsalan-mousavian** [comment]
Addressed the comments. Please take a look.

**Pull request merged**